docs/contributing/contributing-icons.stories.mdx

```diff
@@ -0,0 +1,94 @@
+import { Meta } from '../../.storybook/components';
+
+<Meta title="Introduction/Contributing/Contributing Icons" />
+
+# Contributing Icons
+
+This page outlines the process of contributing an icon to the `@sumup/icons` package.
+
+> Note that icons should be added by internal contributors with access to the SumUp [Figma icons library](https://www.figma.com/file/vnFVuPNlqF45rkw1u9toBC/SumUp-Iconography) (internal link). If you don't have access but would like to see an icon added to `@sumup/icons`, please [open an issue](https://github.com/sumup-oss/circuit-ui/issues/new).
+
+## Adding a new icon to `@sumup/icons`
+
+1. Create a new SVG file for each icon size in [`packages/web/icons/v2/`](https://github.com/sumup-oss/circuit-ui/tree/main/packages/icons/web/v2) with the name `name_size.svg` (e.g. `add_items_24`—this will generate an `<AddItems />` component).
+2. Export the icon as SVG from the [Figma icons library](https://www.figma.com/file/vnFVuPNlqF45rkw1u9toBC/SumUp-Iconography) (internal link). If the icon isn't in the library, make a request with the design team first.
```

Comment:
Huh, I always used the `Export` panel in the right sidebar. This is easier!

Reply:
Yes, I also find it easier than having to find the file in your downloads and move/rename it 😅 Personal preference but that's a way